doc/api/errors.md

```diff
@@ -222,7 +222,7 @@ myObject.stack;  // similar to `new Error().stack`
 ```
 
 The first line of the trace, instead of being prefixed with `ErrorType:
-message`, will be the result of calling `targetObject.toString()`.
+message`, will be formatted as `Error: <targetObject message>`.
```

Comment:
The first line of the stack trace in the sample code immediately above is merely `Error` and nothing else, so I think this change is not correct.  `myObject.toString()` in the sample code returns `[object Object]` and that is not used either, so the current documentation appears incorrect or incomplete as well....